mlt-trans.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/sh
+# Describtion: This scipt is used for CPU  Multi-instance Translate, 
+#              which can  greatly speedup translate performance.
+# FileName: mlt-trans.sh
+# Usage: ./mlt-trans.sh model_name file_to_translate batch_size [benchmark]
+# Version: 2.0
+
+
+# split the real testset file to multi small files for multi-instances
+# param 1: file to translate
+# param 2: numbers of instances
+function split_file(){
+    file=$1
+	workers=$2
+	lines=`cat $1 | wc -l`
+
+	quot=$(($lines/$workers))
+	rema=$(($lines%workers))
+	if [ $rema -ne 0 ];then
+		quot=$(($quot+1))
+		cat $file|head -n $(($quot*$workers-$lines)) >> $file
+	fi
+	split -l $quot $file -d -a 2 mlt-gnmt.log.
+	head -n $lines $file > temp.log
+	mv temp.log $file
+}
+
+# benchmark: inference use dummy data
```

Comment:
what kind of dummy data?

Reply:
we use input file as dummy data but not divided. That is to say each instance translate the whole input file. This is used as benchmark test.